### PR TITLE
Register ees

### PR DESCRIPTION
**Pull request by kdelee**
This is https://github.com/ansible/awx-operator/pull/356 but rebased on devel + with new change to use https://github.com/ansible/awx/pull/10377

### CODE REVIEW

**Comment by Zokormazo** [comment]
Seems good to me. Rerunned the pr check

File: deploy/crds/awx_v1beta1_crd.yaml [hunk 1]
```diff
@@ -151,6 +151,8 @@ spec:
                         type: string
                 control_plane_ee_image:
                   description: Registry path to the Execution Environment container image to use on control plane pods
```
Comment (Zokormazo) [inline deploy/crds/awx_v1beta1_crd.yaml:153]: missing type here

**Review by Zokormazo — DISMISSED** [review]

File: deploy/crds/awx_v1beta1_crd.yaml (hunk 1 above)
Comment (kdelee) [reply deploy/crds/awx_v1beta1_crd.yaml:153]: what is type

Comment (kdelee) [reply deploy/crds/awx_v1beta1_crd.yaml:153]: oh string I guess

File: ansible/templates/crd.yml.j2 [hunk 2]
```diff
@@ -151,6 +151,8 @@ spec:
                         type: string
                 control_plane_ee_image:
                   description: Registry path to the Execution Environment container image to use on control plane pods
```
Comment (kdelee) [inline ansible/templates/crd.yml.j2:153]: @Zokormazo so I think when I did the rebase I messed this up. all these need to have a type, right?

**Pull request merged**